greylist_db.py

```
@@ -0,0 +1,76 @@
+        await db.commit()
+
+# Initialize table synchronously to avoid event loop issues during import
+import sqlite3 as _sl
+with _sl.connect(DB_PATH) as _conn:
+    _conn.execute(CREATE_TABLE_SQL)
+
+SYNC_DB_CONN = sqlite3.connect(DB_PATH, check_same_thread=False)
+SYNC_DB_CONN.execute(CREATE_TABLE_SQL)
+SYNC_DB_CONN.commit()
+
+async def upsert_greylist(email: str, mx_host: str, retry_delay: int, tries: int):
```

Ruff (0.12.2)

Unused function argument: `tries`

(ARG001)